src/main/java/com/replaymod/replay/ReplayModReplay.java

```diff
@@ -8,6 +8,7 @@
 import com.replaymod.core.Module;
 import com.replaymod.core.ReplayMod;
 import com.replaymod.core.utils.ModCompat;
+import com.replaymod.core.utils.Utils;
```

Comment:
Is this import required?

Reply:
No, seems I missed that when I moved some code to a different file